examples/docker/src/aem/default/etc/aem.yml

```diff
@@ -95,9 +95,14 @@ instance:
     # OSGi components state tracking
     component_stable:
       skip: false
-      pids_ignored: []
-      pids_failed_activation: []
-      pids_unsatisfied_reference: []
+      pids:
+        include: []
+        exclude: []
+        match:
```

Comment:
notice that there is no `active` here. it's implicitly calculated which means that you need to only configure "exceptions" - the components that are not active; all others e.g newly developed will be checked for activeness